homeassistant/components/netatmo/climate.py

```diff
@@ -415,6 +417,8 @@ def device_state_attributes(self):
                 "heating_power_request", 0
             )
 
+        attr[ATTR_SELECTED_SCHEDULE] = self._selected_schedule
```

Comment:
We should only add the item to the state attributes if the value is not None.

Reply:
Thanks, no problem. Will open a PR.

Reply:
PR for address this: https://github.com/home-assistant/core/pull/45176